dev-tools/packer/platforms/binary/build.sh

```diff
@@ -8,7 +8,7 @@ ARCHDIR=${BASEDIR}/../../
 # executed from the top directory
 runid=binary-$BEAT-$ARCH
 
-cat ${BUILD_DIR}/package.yml ${ARCHDIR}/archs/$ARCH.yml ${ARCHDIR}/version.yml > ${BUILD_DIR}/settings-$runid.yml
+cat ${BUILD_DIR}/package.yml ${ARCHDIR}/archs/$ARCH.yml > ${BUILD_DIR}/settings-$runid.yml
```

Comment:
They are indirectly required as I added version.yml to package.yml as I already need it in before_build.sh. The alternative would be to create an additional .yml file in before_build but as we always need the version, would only add complexity.